### PR TITLE
Add test for Log Out options in workbenches

### PR DESCRIPTION
**Pull request by bdattoma**
This PR implements ODS-1971

Signed-off-by: bdattoma <bdattoma@redhat.com>

### CODE REVIEW

File: tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource [hunk 1]
```diff
@@ -197,26 +197,34 @@ Start Workbench
 
 Access To Workbench
     [Documentation]    Gets access to a workbench from DS Project details page after it got launched
-    [Arguments]    ${username}=${TEST_USER.USERNAME}  ${password}=${TEST_USER.PASSWORD}  ${auth_type}=${TEST_USER.AUTH_TYPE}
-    Switch Window   NEW
+    [Arguments]    ${username}  ${password}  ${auth_type}
+    # Switch Window   NEW
```
Comment (lugi0) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource:201]: Can you remove this line if it's not needed?

File: tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot [hunk 2]
```diff
@@ -96,11 +96,11 @@ Start JupyterLab Notebook Server
 
 Open JupyterLab Control Panel
   Open With JupyterLab Menu  File  Hub Control Panel
-  #Switch Window  JupyterHub
+  SeleniumLibrary.Switch Window    NEW
 
 Stop JupyterLab Notebook Server
   Open JupyterLab Control Panel
-  SeleniumLibrary.Switch Window    NEW
+  #SeleniumLibrary.Switch Window    NEW
```
Comment (lugi0) [inline tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot:103]: Can you remove this line?

File: tests/Resources/Page/LoginPage.robot [hunk 3]
```diff
@@ -48,3 +48,17 @@ Login To Openshift
     Input Text  id=inputPassword  ${ocp_user_pw}
     Click Element  xpath=/html/body/div/div/main/div/form/div[4]/button
     Maybe Skip Tour
+
+Log In Should Be Requested
+    [Documentation]    Passes if the login page appears and fails otherwise
+    ${present}=    Is OpenShift Login Visible
+    IF    ${present} == ${FALSE}
+        Fail    msg=Log in page did not appear as expected       
+    END
```
Comment (lugi0) [inline tests/Resources/Page/LoginPage.robot:57]: I would do an inline IF but it's fine either way

**Comment by bdattoma** [comment]
Jenkins Job: [1524](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1524/console) _SUCCESS_

Jenkins Job for smoke: _COMING SOON_

File: tests/Resources/Page/LoginPage.robot (hunk 3 above)
Comment (bdattoma) [reply tests/Resources/Page/LoginPage.robot]: done

File: tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot (hunk 2 above)
Comment (bdattoma) [reply tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot]: done

File: tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource (hunk 1 above)
Comment (bdattoma) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource]: done

**Review by lugi0 — DISMISSED** [review]

**Review by jgarciao — APPROVED** [review]

**Review by lugi0 — APPROVED** [review]

**Pull request merged**